vcf.c

```diff
@@ -3544,11 +3635,10 @@ int vcf_parse(kstring_t *s, const bcf_hdr_t *h, bcf1_t *v)
     // almost certainly is never detected by the compiler so has no impact,
     // but equally so this code has minimal (often beneficial) impact on
     // performance too.)
-    s->s[s->l+0] = 0;
-    s->s[s->l+1] = 0;
-    s->s[s->l+2] = 0;
-    s->s[s->l+3] = 0;
-
+    for(int i = 0; i < 4; i++)
```

Comment:
Functionally same as original, we should probably remove this.

Reply:
Those were running in the original but the we way we use it in the parse_info functions breaks it. I'll make sure to remove those.